tests/app/main/views/test_index.py

```diff
@@ -35,7 +43,28 @@ def it_should_show_future_events_in_cards(self, client, sample_future_events, sa
 
         assert content == intro_course['title']
 
-    @pytest.mark.parametrize('div_class', ['.navbar', '.footnav'])
+    def it_should_display_text_for_main_article(self, mocker, client, sample_future_events, sample_articles_summary):
+        mocker.patch('app.main.views.index.randint', return_value=0)
+        response = client.get(url_for(
+            'main.index'
+        ))
+        page = BeautifulSoup(response.data.decode('utf-8'), 'html.parser')
+        title = page.select_one("#main_article h2").text
+        content = page.select_one("#main_article p").text
+
+        assert title == sample_articles_summary[0]['title']
+        assert content == sample_articles_summary[0]['short_content'] + " READ MORE"
+
+    def it_should_show_featured_articles_in_cards(self, client, sample_future_events, sample_articles_summary):
+        response = client.get(url_for(
+            'main.index'
+        ))
+        page = BeautifulSoup(response.data.decode('utf-8'), 'html.parser')
+        content = page.find_all(["h4"])[6].string
```

Comment:
This is a little bit confusing, not sure where `6` comes from, would be better to add in identifiers for the `div` section and do a css selector on that to identify the articles. 

Also the var name `content` doesn't appear to match the assert, I would suggest changing it to `article_title` to make the test easier to read